model/model.pb.go

```diff
@@ -1,36 +1,16 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: model.proto
 
-/*
-	Package model is a generated protocol buffer package.
-
-	It is generated from these files:
-		model.proto
-
-	It has these top-level messages:
-		KeyValue
-		Log
-		SpanRef
-		Process
-		Span
-		Trace
-		Batch
-		PostSpansRequest
-		PostSpansResponse
-		GetTraceRequest
-		GetTraceResponse
-*/
 package model
 
 import proto "github.com/gogo/protobuf/proto"
 import golang_proto "github.com/golang/protobuf/proto"
 import fmt "fmt"
 import math "math"
-import _ "github.com/gogo/protobuf/types"
-import _ "github.com/gogo/protobuf/types"
 import _ "github.com/gogo/googleapis/google/api"
-import _ "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/options"
 import _ "github.com/gogo/protobuf/gogoproto"
+import _ "github.com/gogo/protobuf/types"
```

Comment:
it looks like you're using different version of the generator, there should not be so many changes if you only add one field

Reply:
I am getting these diffs too, need to track them down. I see gogo version was changed in glide.lock in 933efb3a